### PR TITLE
chore: Add prefetch dns, add loading indicator to dashboard

### PR DESCRIPTION
**Pull request by istarkov**
## Description

1. What is this PR about (link the issue and add a short description)

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Review by kof — APPROVED** [review]

File: apps/builder/app/dashboard/projects/project-card.tsx [hunk 1]
```diff
@@ -180,11 +179,30 @@ export const ProjectCard = ({
...
+  useEffect(() => {
+    const linkPath = builderUrl({ origin: window.origin, projectId: id });
+
+    const handleNavigate = (event: NavigateEvent) => {
+      if (event.destination.url === linkPath) {
+        setIsTransitioning(true);
+      }
+    };
+
+    if (window.navigation === undefined) {
```
Comment (TrySound) [inline apps/builder/app/dashboard/projects/project-card.tsx:193]: is it not available somewhere?

Comment (istarkov) [reply apps/builder/app/dashboard/projects/project-card.tsx:193]: my safari gives undefined

Comment (istarkov) [reply apps/builder/app/dashboard/projects/project-card.tsx:193]: will recheck

Comment (istarkov) [reply apps/builder/app/dashboard/projects/project-card.tsx:193]: image

File: apps/builder/app/routes/_ui.dashboard.tsx [hunk 2]
```diff
@@ -90,9 +95,14 @@ const DashboardRoute = () => {
   const data = useLoaderData<typeof loader>();
 
   return (
-    <ClientOnly>
-      <Dashboard {...data} />
-    </ClientOnly>
+    <>
+      {data.projects.map((project) => {
+        prefetchDNS(builderUrl({ projectId: project.id, origin: data.origin }));
```
Comment (TrySound) [inline apps/builder/app/routes/_ui.dashboard.tsx:100]: no return?

Comment (TrySound) [reply apps/builder/app/routes/_ui.dashboard.tsx:100]: why not just for/of?

Comment (istarkov) [reply apps/builder/app/routes/_ui.dashboard.tsx:100]: void, react magic

**Pull request merged**